src/pyodide/internal/python.ts

```diff
@@ -62,10 +62,10 @@ async function prepareWasmLinearMemory(Module: Module): Promise<void> {
   mountSitePackages(Module, SITE_PACKAGES.rootInfo);
   entropyMountFiles(Module);
   Module.noInitialRun = !SHOULD_RESTORE_SNAPSHOT;
-  preloadDynamicLibs(Module);
-  Module.removeRunDependency('dynlibs');
+  enterJaegerSpan('preloadDynamicLibs', () => preloadDynamicLibs(Module));
```

Comment:
Nit: this likely doesn't matter all that much but our convention so far has been snake_case for these